doc/source/reference/ufuncs.rst

```diff
@@ -228,45 +228,45 @@ can generate this table for your system with the code given in the Figure.
 
 .. admonition:: Figure
 
-    Code segment showing the "can cast safely" table for a 32-bit system.
+    Code segment showing the "can cast safely" table for a 64-bit system.
+    Generally the output depends on the system; your system might result in
+    a different table.
 
+    >>> mark = {False: ' -', True: ' ✓'}
     >>> def print_table(ntypes):
-    ...     print 'X',
-    ...     for char in ntypes: print char,
-    ...     print
+    ...     print('X ' + ' '.join(ntypes))
     ...     for row in ntypes:
-    ...         print row,
+    ...         print(row, end='')
     ...         for col in ntypes:
-    ...             print int(np.can_cast(row, col)),
-    ...         print
-    >>> print_table(np.typecodes['All'])
```

Comment:
Sorry, did this line vanish completely? I had missed that.

Reply:
Oops, fixed.  I'll stop force-pushing until everything looks right.

Reply:
I honestly do not care for this small change. Lets juts get it done :).